tests/testthat/test-utils-parse.R

```diff
@@ -327,12 +327,39 @@ test_that('parseWaterQualityMeasurements doesnt error when given invalid JSON',
                             "timeZone": "CST"
                             }
                             ]
-}')
+  }')
 
   reportObject2 <- fromJSON('{ }')
   
   expect_equal(repgen:::parseWaterQualityMeasurements(reportObject1), NULL)
   expect_equal(repgen:::parseWaterQualityMeasurements(reportObject2), NULL)
-  })
+})
+
+test_that('fetchExcludedControlConditions properly retrieves the excluded control condition data', {
```

Comment:
Whoops, good catch. I just copied the fetch test over to be the read test but forgot to update the name